grabber/main.go

```diff
@@ -328,15 +328,16 @@ func checkAncestors(ctx context.Context, importer *backend.Backend, blockNumber
 }
 
 // ensureTxsConsistent checks if the block tx count matches the number of txs in the db, and reimports the block if not.
-// If a block is reimported, then the parent hash is returned.
-func ensureTxsConsistent(ctx context.Context, importer *backend.Backend, blockNumber int64, checkTxCount bool) (common.Hash, error) {
-	if ok, err := importer.TransactionsConsistent(blockNumber); err != nil {
+// If a block is reimported and the parent changed, then the parent hash is returned.
```

Comment:
This should be an efficiency improvement, since the sole caller was assuming this already.